docs/spec/metricsets/metricset.json

```diff
@@ -20,6 +20,61 @@
                             "$ref": "sample.json"
                         }
                     },
+                    "properties": {
+                        "transaction": {
+                            "type": ["object", "null"],
+                            "properties": {
+                                "duration": {
+                                    "type": [
+                                        "object",
+                                        "null"
+                                    ],
+                                    "properties": {
+                                        "count": {
+                                            "type": "number"
+                                        },
+                                        "sum.us": {
+                                            "type": "number"
+                                        }
+                                    }
+                                },
+                                "breakdown.count": {
```

Comment:
i guess this is not ok, and I need to create a `breakdown` object with `count` inside, right?

Reply:
Can you try it out with sample requests using `breakdown.count:` and `breakdown: { count: }` notation?